src/input.cpp

```diff
@@ -1189,6 +1189,22 @@ const std::string &input_context::handle_input( const int timeout )
     next_action.type = input_event_t::error;
     const std::string *result = &CATA_ERROR;
     while( true ) {
+
+        //Allows "toggle_language_to_en" to also work on contexts other than "DEFAULTMODE"
+        if( !next_action.sequence.empty() ) {
+            auto attr = inp_mngr.get_action_attributes( "toggle_language_to_en", "DEFAULTMODE" );
+            if( !attr.input_events.empty() ) {
+                input_event input = attr.input_events.front();
```

Comment:
I don't think copying the input event object is really necessary.

Reply:
const& would be ok, right?